app/controllers/rides_controller.rb

```diff
@@ -1,2 +1,37 @@
 class RidesController < ApplicationController
+
+  before_action :find_ride, only: [:show, :update]
+
+  def new
+    @ride = Ride.new
+  end
+
+  def show
+  end
+
+  def create
+    ride = Ride.new(ride_params)
+
+    if ride.save
+      redirect_to ride_path(ride.id)
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.